README.md

```diff
@@ -0,0 +1,3 @@
+js-to-firepath converter
+
+-converts normal syntax expression into fhirpath expression
```

Comment:
This is where you document how to use it, from installation to importing the function to using the function.  Document what "normal syntax" is, and include some sample input and output.